tests/integration/security/main_test.go

```diff
@@ -75,7 +75,12 @@ meshConfig:
       service: %q
       port: 8000
       pathPrefix: "/check"
-      includeHeadersInCheck: ["x-ext-authz"]
+      headersToUpstreamOnAllow: ["x-ext-authz-*"]
```

Comment:
nit: we may want to start using istio.PatchMeshConfig now. This was a bad idea before since mesh config changes took ~2 mins to propagate, now they should be instance

Reply:
thanks, will update later.